Sources/SwiftlySearch/SwiftlySearch.swift

```diff
@@ -106,10 +108,9 @@ struct SearchBar<ResultContent: View>: UIViewControllerRepresentable {
 
         // MARK: - UISearchResultsUpdating
         func updateSearchResults(for searchController: UISearchController) {
-            guard let text = searchController.searchBar.text else { return }
-            // Make sure the text has actually changed (workaround for #10).
-            guard text != self.text else { return }
+            guard let text = searchController.searchBar.text, updatedText != text else { return }
```

Comment:
I would prefer if this check stayed as two guards, for increased readability.
```suggestion
            guard let text = searchController.searchBar.text else { return }
            // Make sure the text has actually changed (workaround for #10).
            guard updatedText != text else { return }
```

Reply:
Implemented in faffe21e489ba7f3830e2f7b2be3963f92d99375